Source/Docs/settings-options.rst

```diff
@@ -13,6 +13,7 @@ Compiler Settings
 
     Compiler.Platform,"Platform x86, x64, ARM32, ARM64"
     Compiler.BaseAddress,Base address of the compiled application
+	Compiler.InitialStackAddress,Initial address of the stack
```

Comment:
Currently all code using this config assumes that it points to the top of the stack. I think it would be good to document explicitly that this address should be the top of the stack and not the bottom.